test/check-application

```diff
@@ -253,6 +253,49 @@ class TestApplication(testlib.MachineCase):
 
         self.allow_journal_messages("/run/podman/io.podman: couldn't connect.*")
 
+    def testRunImage(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/podman")
+        b.wait_present("#containers-images")
+        b.wait_in_text("#containers-images", "busybox:latest")
+
+        # Open run image dialog
+        b.wait_present('#containers-images tr:contains("busybox:latest")')
```

Comment:
This should wait on the button to appear, just in case.

Reply:
Done